addons/l10n_at/demo/demo_company.xml

```diff
@@ -18,6 +18,13 @@
     <field name="partner_id" ref="partner_demo_company_at"/>
   </record>
 
+  <record id="demo_bank_at" model="res.partner.bank">
+    <field name="acc_number">AT285400035566419993</field>
+    <field name="bank_id" ref="base.bank_ing"/>
```

Comment:
remove all `bank_id` because it's not required and `bank_ing` only work in belgium.